.Rbuildignore

```diff
@@ -1,3 +1,5 @@
+^renv$
+^renv\.lock$
```

Comment:
Can you describe briefly why these were changed?

Reply:
Hey, I just added these mainly because I thought the renv didn't need to be a part of the package build - just as you wouldn't want it being pushed up to github - is that not correct? I can change it back if needed

Reply:
That is good. Just wanted to be sure what was the purpose.

Reply:
Ok nice sounds good.